docs/documentation/angular-workspace.md

```diff
@@ -6,7 +6,7 @@
 
 - **version** (`integer`): File format version. This is currently `"1"`.
 
-- **newProjectRoot** (`string`): Path where new projects will be created.
+- **newProjectRoot** (`string`): (**DEPRECATED**) Path where new projects will be created.
```

Comment:
this looks wrong, shouldn't it be on the following option?

Reply:
Jeez, that's what I get for amending commits in a rush.